vhpc_toolkit/operations.py

```diff
@@ -235,6 +228,125 @@ def _get_clone_task(self, vm_cfg):
             )
             raise SystemExit
 
+    @staticmethod
+    def _create_resource_pool(resource_pool_name, destination_host: vim.HostSystem):
+        resource_spec = vim.ResourceConfigSpec()
+        resource_allocation_info = vim.ResourceAllocationInfo()
+        resource_allocation_info.expandableReservation = True
+        resource_allocation_info.reservation = 0
+        resource_allocation_info.limit = -1
+        resource_allocation_info.shares = vim.SharesInfo()
+        resource_allocation_info.shares.level = vim.SharesInfo.Level.normal
+        resource_spec.cpuAllocation = resource_allocation_info
+        resource_spec.memoryAllocation = resource_allocation_info
```

Comment:
line 240 and 241 use force conversion?

Reply:
Add a warning log output "Use default resource creation policy"

Reply:
self.logger.warning("Use default policy of resource pool creation. No limitation on CPU and memory allocation.")